.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing
+
+There's no such thing as a small contribution, anyway you can help improve our website is welcome. All we ask is that we are a volunteer of [Women Hack For Non-Profits](http://www.womenhackfornonprofits.com/volunteer-contact.html) and you identify yourself as a woman.
+
+### Who are [Women Hack For Non-Profits](http://www.womenhackfornonprofits.com/)
+Women Hack for Non-Profits matchmakes skilled women in technology to open source projects for non-profits, charities and individuals with a good social cause. By doing so, we aim to enhance the skill set of Women in Tech to support their growth and increase their footprint in the open source community.
+
+### How can I contribute?
```

Comment:
Did we have any guidelines in terms if code style/what sort of changes are okay/not okay to do?